src/Data/filterSchema.ts

```diff
@@ -0,0 +1,52 @@
+import { pisaClient } from './pisaClient'
+import { DataClassAttributes } from './types'
+
+/**
+ * Fetches the schema for `subPath`.
+ * Filters out boolean and reference attributes, since they are currently not fully supported.
+ * It also converts enum values, if in a wrong format.
+ **/
+export async function filterSchema(subPath: string) {
+  const client = await pisaClient(subPath)
+  const schema = (await client.get('schema')) as {
+    attributes: DataClassAttributes
+  }
+
+  return Object.fromEntries(
+    Object.entries(schema.attributes)
+      .filter(
+        ([_, attribute]) => !['boolean', 'reference'].includes(attribute[0]),
+      )
+      .map(([key, value]) => [
+        key,
+        value[0] === 'enum'
+          ? ['enum', correctWrongEnumValues(value[1])]
+          : value,
+      ]),
+  )
+}
+
+type LocalizedEnumValueConfig = {
+  value: string
+  title: string
+}
+
+type EnumValueConfig = string | LocalizedEnumValueConfig
+
+function correctWrongEnumValues({
```

Comment:
* [ ] Why do we need this? According to the Scrivito SDK type there's nothing wrong with string-only values. Am I missing something? https://github.com/Scrivito/scrivito_sdk_js/blob/1446f9ab9a89c0ec105f3cdd03b130d259bae3a2/src/scrivito_sdk/data_integration/data_class_schema.ts#L75